gpu-deployment-tutorial/gpu-deployment-from-scratch.md

```diff
@@ -0,0 +1,353 @@
+
+#### Launching a Brev VM
+
+- Under "GPUs" select "New Instance"
+- Choose a GPU type that costs <$1/hour (e.g an L4) 
+- Choose any provider
+- Give your VM a name
+- Hit Deploy
+
+#### Connecting to your VM
+
+Once your VM is deployed, follow the Brev access instructions provided for your instance. The connection instructions will vary depending on your operating system. For example, on macOS you would:
+
+- Install the `brev` CLI
+  - `brew install brevdev/homebrew-brev/brev 
+- Login to your account (copy from access page)
+  - `brev login --token ****`
+- Connect via SSH
+  - `brev shell <your vm name>`
+
+For Linux and Windows instructions check the [brev-cli install documentation](https://docs.nvidia.com/brev/latest/brev-cli.html#installation-instructions)
+
+### Exploring our GPU Software Environment
+
+Let's start by exploring our VM to see what software we got out of the box.
+
+```console
+$ cat /etc/os-release
+PRETTY_NAME="Ubuntu 22.04.5 LTS"
+```
+
+We can check our GPU information by running `nvidia-smi`.
+
+```console
+$ nvidia-smi
++-----------------------------------------------------------------------------------------+
+| NVIDIA-SMI 570.158.01             Driver Version: 570.158.01     CUDA Version: 12.8     |
+|-----------------------------------------+------------------------+----------------------+
+| GPU  Name                 Persistence-M | Bus-Id          Disp.A | Volatile Uncorr. ECC |
+| Fan  Temp   Perf          Pwr:Usage/Cap |           Memory-Usage | GPU-Util  Compute M. |
+|                                         |                        |               MIG M. |
+|=========================================+========================+======================|
+|   0  NVIDIA L4                      On  |   00000000:00:03.0 Off |                    0 |
+| N/A   47C    P8             13W /   72W |       0MiB /  23034MiB |      0%      Default |
+|                                         |                        |                  N/A |
++-----------------------------------------+------------------------+----------------------+
+
++-----------------------------------------------------------------------------------------+
+| Processes:                                                                              |
+|  GPU   GI   CI              PID   Type   Process name                        GPU Memory |
+|        ID   ID                                                               Usage      |
+|=========================================================================================|
+|  No running processes found                                                             |
++-----------------------------------------------------------------------------------------+
+```
+
+### Let's install something 
+
+```console
+$which pip
+/usr/bin/pip
+```
+
+If you have pip, we can try to install something like `cupy` 
```

Comment:
@jacobtomlinson I tried installing `cudf` and it looks like cudf from pip brings everything it needs, I was able to install things and run operations. it's also trick, because if you install cudf import cudf first and then cupy, it will work. 

So to force an error I opted for cupy, becasue I know it misses `libnvrtc`. At least until the next release.